modules/playing-cards/widgets/cards-widget.php

```diff
@@ -0,0 +1,312 @@
+					'2' => esc_html__( '2', 'elementor' ),
+					'3' => esc_html__( '3', 'elementor' ),
+					'4' => esc_html__( '4', 'elementor' ),
+					'5' => esc_html__( '5', 'elementor' ),
+					'6' => esc_html__( '6', 'elementor' ),
+					'7' => esc_html__( '7', 'elementor' ),
+					'8' => esc_html__( '8', 'elementor' ),
+					'9' => esc_html__( '9', 'elementor' ),
+					'10' => esc_html__( '10', 'elementor' ),
+					'J' => esc_html__( 'J', 'elementor' ),
+					'Q' => esc_html__( 'Q', 'elementor' ),
+					'K' => esc_html__( 'K', 'elementor' ),
+				],
+			]
+		);
+
+		$card_suits_options = array_map( function( $option) {
+			return $option['symbol'];
+		}, $this->card_suites_map);
+
+		$repeater->add_control(
+			'card_suit',
+			[
+				'label' => esc_html__( 'Suits', 'elementor' ),
+				'type' => Controls_Manager::SELECT,
+				'default' => 'clubs',
+				'options' => $card_suits_options
+			]
+		);
+		
+		$this->add_control(
+			'card_list',
+			[
+				'label' => esc_html__( 'Cards', 'elementor' ),
+				'type' => Controls_Manager::REPEATER,
+				'fields' => $repeater->get_controls(),
+				'title_field' => '{{{ card_value }}} of {{{ card_suit }}}',
+
+			]
+		);
+
+		$this->add_control(
+			'card_back_image',
+			[
+				'label' => esc_html__( 'Choose back image', 'elementor' ),
+				'type' => Controls_Manager::MEDIA,
+				'dynamic' => [
+					'active' => true,
+				],
+			]
+		);
+
+		$this->add_responsive_control(
+			'align_cards',
+			[
+				'label' => esc_html__( 'Alignment', 'elementor' ),
+				'type' => Controls_Manager::CHOOSE,
+				'options' => [
+					'left'    => [
+						'title' => esc_html__( 'Left', 'elementor' ),
+						'icon' => 'eicon-text-align-left',
+					],
+					'center' => [
+						'title' => esc_html__( 'Center', 'elementor' ),
+						'icon' => 'eicon-text-align-center',
+					],
+					'right' => [
+						'title' => esc_html__( 'Right', 'elementor' ),
+						'icon' => 'eicon-text-align-right',
+					],
+				],
+				'default' => 'center',
+				'selectors' => [
+					'{{WRAPPER}} .elementor-card-list-items' => 'justify-content: {{VALUE}}',
+				],
+			]
+		);
+
+
+		$this->end_controls_section();
+
+		$this->start_controls_section(
+			'section_card_style',
+			[
+				'label' => esc_html__( 'Cards', 'elementor' ),
+				'tab' => Controls_Manager::TAB_STYLE,
+			]
+		);
+
+		$this->add_responsive_control(
+			'playing_card_item_space_between',
+			[
+				'label' => esc_html__( 'Space between cards', 'elementor' ),
+				'type' => Controls_Manager::SLIDER,
+				'size_units' => [ 'px' ],
+				'range' => [
+					'px' => [
+						'min' => 0,
+						'max' => 200,
+					],
+				],
+				'default' => [
+					'size' => 12,
+				],
+				'selectors' => [
+					'{{WRAPPER}}' => '--e-playing-cards-gap: {{SIZE}}{{UNIT}}',
+				],
+			]
+		);
+
+		$this->add_responsive_control(
+			'playing_card_item_symbol_size',
+			[
+				'label' => esc_html__( 'Symbol size', 'elementor' ),
+				'type' => Controls_Manager::SLIDER,
+				'size_units' => [ 'px' ],
```

Comment:
Same as above

Reply:
specifically here i prefer to keep only px as the option so it will fit the card size properly